### ISSUE TITLE
scene terminology

### ISSUE THREAD

**Issue by mfiano** [HackerTheory/first-light#6]
scenes can have these operators on them:

~~PREPARE - load the scene-definition file into core-state, load lisp files/extension associated with it.~~

~~LOAD - take a named prepared scene, run initializers, add all actors to current scene.~~

REPLACE - Same as load, but first remove all current actors, then load the named scene.

UNLOAD - remove all actors and components in the named scene and currently loaded scene.

If you load multiple scenes, they all merge into the current scene-tree.
unloading a scene gets rid of just those actors that were a part of that scene.

Need to add a slot to actor indicating what scene it is in. there is always the ":default" scene which if unloaded, all actors are removed, but the scene is still present.

The concept of :default scene, current scene, etc, and how they differ are still a little nebulous.

**Comment by psilord** [comment]
This is blocked by #12.
